frontend/packages/data-portal/app/components/Filters/DropdownFilterButton.tsx

```diff
@@ -62,8 +62,8 @@ export function DropdownFilterButton({
             'group-hover:!fill-black transition-colors',
             open ? '!fill-black' : '!fill-sds-gray-500',
           )}
-          sdsIcon="chevronDown"
-          sdsSize="s"
```

Comment:
This size change was intentional?

Reply:
@codemonkey800 Use XS -- this is the size used for the minimal dropdown component in [SDS's documentation](https://sds.czi.design/009eaf17b/p/1004b1-dropdown-input/t/page-1004b1-73023306-993ff3-17)